app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  layout 'standard'
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(params.require(:comment).permit(:text, :user_id, :post_id))
+    @comment.user = current_user
+    if @comment.save
+      flash[:sucess] = 'Comment created successfuly!'
+      redirect_to user_post_path(id: @comment.post_id, user_id: @comment.user_id)
+    else
+      flash.now[:error] = 'Error, the comment cannot be created!'
+      render :new, locals: { comment: @comment }
+    end
+  end
+end
```

Comment:
# Optional
- I advise you to add `params` method because it is a better security practice to help prevent accidentally allowing users to update sensitive model attributes.
- [Lesson link about the strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)
👇 
```ruby
 private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```

Reply:
✅ Done

Thanks for your suggestion..